.github/workflows/on-issue-creation.yml

```diff
@@ -15,9 +15,10 @@ jobs:
           project-url: https://github.com/orgs/grafana/projects/119
           github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
 
-  add-latest-version-comment:
-    name: Add latest version comment to issue
+  add-latest-version-comment-to-feature-request-issues:
+    name: Add latest version comment to feature request issues
     runs-on: ubuntu-latest
+    if: contains(github.event.issue.labels.*.name, 'feature request')
```

Comment:
no need to add this comment to bug issues, as those have the required input field for OnCall version